bootstrap/kubeadm/types/upstreamv1beta1/types.go

```diff
@@ -107,8 +107,8 @@ type ClusterConfiguration struct {
 	CertificatesDir string `json:"certificatesDir,omitempty"`
 
 	// ImageRepository sets the container registry to pull images from.
```

Comment:
Let's not modify the old kubeadm structs. (this triggers changes in v1alpha3 API types which are basically frozen)